.github/workflows/unit-test.yml

```diff
@@ -28,7 +28,7 @@ jobs:
         strategy:
             fail-fast: false
             matrix:
-                node: ['14']
+                node: ['18']
```

Comment:
The only problem is with the `engines` setting in the root `package.json`. It pins both Node and npm to the latest version so that we can ensure developers have the same local environment. However, CI will fail when running Node v16 because we have `engine-strict` set to true. Perhaps we can ignore this setting on CI with the `--engine-strict=false` CLI arg? WDYT?

Reply:
That would be great to have it possible to run with different Node.js versions.

Reply:
Maybe the GitHub action supports it, see how the registry URL gets passed here:

https://github.com/WordPress/gutenberg/blob/trunk/.github/workflows/build-plugin-zip.yml#L329L333

Follow-up sounds great 👍

Reply:
I don't see any useful option in the GitHub action for this tweak:

https://github.com/actions/setup-node/blob/main/action.yml

We will figure out a different approach.